harbor

```diff
@@ -57,6 +57,7 @@ export DOCKER_PHP_VERSION=${DOCKER_PHP_VERSION:-7.3}
 export DOCKER_POSTGRES_VERSION=${DOCKER_POSTGRES_VERSION:-11}
 export DOCKER_NODE_VERSION=${DOCKER_NODE_VERSION:-12}
 export DOCKER_PHP_XDEBUG=${DOCKER_PHP_XDEBUG:-off}
+export DOCKER_SSH_PATH=${DOCKER_SSH_PATH:-./docker/php/ssh}
```

Comment:
In the future we can ask user to interactively provide the path for the ssh, what do you think?